update.py

```diff
@@ -64,7 +65,9 @@ def check_notices(session, sessionData):
             r = session.get(notice['attachment_url'], stream=True)
             r.raw.decode_content = True
             hash_ = hashlib.md5()
-            for chunk in iter(lambda: r.raw.read(4096), b""):
+            notice['attachment_raw'] = ""
```

Comment:
Hmm, this seems to be appending bytes to a string. Have you checked whether this actually works? I think it's better to use an explicit byte stream?

Reply:
> Have you checked whether this actually works?

Yup, it does. By appending the bytes to the string, we're getting back what we would have got anyway previously from `r.raw` 😄